package.json

```
@@ -272,8 +272,8 @@
     "source-map-support": "^0.5.20",
     "strip-bom": "^5.0.0",
     "strip-json-comments": "^4.0.0",
-    "tempy": "^2.0.0",
-    "typescript": "^4.6.3",
+    "tempy": "^3.0.0",
+    "typescript": "^4.7.4",
```

```suggestion
    "typescript": "4.7.x",
```

This being a range in the first place is a mistake - [TypeScript doesn't follow semver](https://github.com/microsoft/TypeScript/issues/14116#issuecomment-280592571) so has a habit of releasing breaking changes as minors.